### ISSUE TITLE
learn_rate and  checkpoint training

### ISSUE THREAD

**Issue by cena001plus** [PaddlePaddle/PaddleDetection#5148]
The default learning rate is 0.4, picodet, 4 cards, so it is set to 0.1, but during training, it is found that the learning rate decreases very slowly, and finally stabilizes at 0.09986, the map improvement becomes slower, and it is still far from the theoretical value, so I plan to Reduce the learning rate and retrain from the checkpoint, but it takes 6 hours to load hundreds of thousands of training data each time I retrain to actually start i training, but I found the example you gave very vague,

-r output/xxxxxxx/1000 ,

 what is the 1000 in here? My saved model starts at 0. Explain whether the documentation can be more serious.

**Comment by cena001plus** [comment]
最大的epoch是300，你这里写1000，这是告诉其他人，这个1000是是其他参数吗？ 不是epoch？  这里只能写数字吗？ best_model.pdparams可以否？    best_model.pdopt可以否？

**Comment by cena001plus** [comment]
框架中的数据检查耗时，不敢随便开启训练啊，5个小时加载数据，两小时训练一个epoch，7、8小时后才知道训练的对不对，这个成本很高啊。

**Comment by heavengate** [comment]
> 最大的epoch是300，你这里写1000，这是告诉其他人，这个1000是是其他参数吗？ 不是epoch？ 这里只能写数字吗？ best_model.pdparams可以否？ best_model.pdopt可以否？

1000是个示例，指你需要resume哪个pdparams前面的序号，可以是best_model

**Comment by heavengate** [comment]
> 框架中的数据检查耗时，不敢随便开启训练啊，5个小时加载数据，两小时训练一个epoch，7、8小时后才知道训练的对不对，这个成本很高啊。

PaddleDetection训练不需要把所有数据加载进来，是在训练过程中顺序读取的，应该不需要5个小时加载数据
